### ISSUE TITLE
Support Python3

### ISSUE THREAD

**Issue by Endle** [mikeboers/MultiMap#2]
Excuse me,
does this package support python3?
Thanks

**Comment by mikeboers** [comment]
No, it does not. I suspect it would be rather tricky to support both in the same codebase. I'll leave this open in case someone wants to try.

**Comment by Endle** [comment]
Would you please write a simple unit test for that?
With this testcase, I'm glad to port it to python3.
(Maybe we can make a fork of current version)

2015-03-06 0:17 GMT+08:00 Mike Boers notifications@github.com:

> No, it does not. I suspect it would be rather tricky to support both in
> the same codebase. I'll leave this open in case someone wants to try.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mikeboers/MultiMap/issues/2#issuecomment-77394092.

## 

Have a nice day!
Zhenbo Li

**Comment by mikeboers** [comment]
It has extensive doctests, which are run by nose if you run the module like a script.

The issue is more that there are API changes with mappings from Python2 to Python3. Would likely need a global PY3 variable, and conditional definition of methods depending on which version it is.

Unfortunately I'm kinda swamped right now. Sorry.

**Comment by Endle** [comment]
<s>I'm glad to maintain a fork of Python3.
Do you mind if I send a new package to pypi then call it MultiMap3?</s>
I've created a new project https://github.com/Endle/momap

**Comment by mikeboers** [comment]
You are welcome to fork MultiMap if you wish to do so.

Regardless, good luck with the new project!
